stories/documentation/forms/fields/checkbox/angular/checkboxfield.stories.ts

```diff
@@ -47,7 +47,8 @@ export const Basic: StoryObj<CheckboxInputComponent & FormFieldComponent> = {
 				argTypes,
 			)}>
 	<lu-checkbox-input ${generateInputs(inputArgs, argTypes)}
-	[(ngModel)]="example"/>
+	[(ngModel)]="example"
+	[mixed]="${mixed}"/>
```

Comment:
Removing this woule be better, as `generateInputs` will take care of that if you just let `mixed` in `inputArgs`.